src/test/java/net/snowflake/client/log/JDK14LoggerWithClientLatestIT.java

```diff
@@ -89,4 +90,60 @@ public void testJDK14LoggerWithQuotesInMessage() {
     logger.debug("Returning column: 12: a: Group b) Hi {Hello 'World' War} cant wait");
     JDK14Logger.setLevel(Level.OFF);
   }
+
+  @Test
+  public void testJDK14LoggingWithMissingLogPathClientConfig() {
+    Path configFilePath = Paths.get("config.json");
+    String configJson = "{\"common\":{\"log_level\":\"debug\"}}";
+    try {
+      Files.write(configFilePath, configJson.getBytes());
+      Properties properties = new Properties();
+      properties.put("client_config_file", configFilePath.toString());
+      Connection connection = getConnection(properties);
+      connection.createStatement().executeQuery("select 1");
+
+      String homePath = systemGetProperty("user.home");
+      Path homeLogPath = Paths.get(homePath, "jdbc");
+      File file = new File(homeLogPath.toString());
+      assertTrue(file.exists());
+
+      Files.deleteIfExists(configFilePath);
+      FileUtils.deleteDirectory(new File(homeLogPath.toString()));
+    } catch (IOException e) {
```

Comment:
we only fail on IOException or SQLException? what about other exceptions? can we just let the exception to be catch by junit and mark the test as failed?

Reply:
any exception causes the fail - let the junit to handle the method when exception is thrown